### ISSUE TITLE
software automatic mouth as NVDA addon

### ISSUE THREAD

**Issue by joshknnd1982** [nvaccess/nvda#14597]
Could you please either build software automatic mouth, sam, into NVDA as another tts option. or make software automatic mouth as an NVDA addon? its source code is here. 
https://discordier.github.io/sam/

**Comment by josephsl** [comment]
Hi,

It is best described as abandonware (by the current maintainer of the source code), therefore it cannot be bundled into NVDA as an add-on unless we are wiling to walk the "copyright gray area" territory.

Thanks.

**Comment by joshknnd1982** [comment]
Well then its best not to walk the grey area since nvAccess is a recognized company that should conform to current laws.
